shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/SQLStatementAssert.java

```diff
@@ -49,6 +56,9 @@
      */
     public static void assertIs(final SQLCaseAssertContext assertContext, final SQLStatement actual, final SQLParserTestCase expected) {
         ParameterMarkerAssert.assertCount(assertContext, actual.getParameterCount(), expected.getParameters().size());
+        if (!expected.getComments().isEmpty()) {
```

Comment:
@tuichenchuxin Can we add a `CommentAssert` to handle this logic? And move `if (!expected.getComments().isEmpty())` into the `assertComment` method.